lua/wikis/commons/Widget/Match/ButtonBar.lua

```diff
@@ -65,11 +90,24 @@ function MatchButtonBar:render()
 				streams = StreamLinks.filterStreams(match.stream),
 				matchIsLive = match.phase == 'ongoing',
 			} or nil,
-			displayVods and VodsDropdown{
-				match = match,
-			} or nil
+			makeDropdownForVods and VodsDropdownButton{count = #vods} or nil,
+			showInlineVods and VodButton{vodLink = vods[1].vod} or nil
```

Comment:
Shouldn't `showInlineVods` rather be a check whether it's a match vod?
Alternative, also pass the vod number here.
E.g. in case there exists only a VOD for G1, it would still make sense to display the number.

Reply:
makes sense